pkg/engine/models/spec.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+// Spec represent the KCL compile result
+type Spec struct {
```

Comment:
what's the abstraction of Spec for ? I don't see the necessity

Reply:
Spec means all desired states of resources in a stack